docs/samples/configuration/README.md

```diff
@@ -0,0 +1,11 @@
+# Configuration Management Documentation
```

Comment:
This should be titled "Configuration Management Samples", because we do have a docs directory under docs/symphony-book/configuration-management. We should add a link from the _overview.md under that folder to this folder.

Reply:
That makes sense, I made these updates.